intercom/service/tag.py

```diff
@@ -5,10 +5,11 @@
 from intercom.api_operations.find import Find
 from intercom.api_operations.find_all import FindAll
 from intercom.api_operations.save import Save
+from intercom.api_operations.delete import Delete
 from intercom.service.base_service import BaseService
 
 
-class Tag(BaseService, All, Find, FindAll, Save):
+class Tag(BaseService, All, Find, FindAll, Save, Delete):
```

Comment:
looks good:)

Reply:
@ellakz could you grant me write access so I could merge this?

Reply:
Thanks! Granted.

Reply:
@ellakz thanks! strangely it didn't get through. I still might not have the write access

Reply:
@apassant hello! Could you help us review on this change? :D

Reply:
I think you need access to intercom/python-intercom to merge...